modules/core/src/lib/picking/pick-info.ts

```diff
@@ -38,6 +38,12 @@ export interface PickingInfo {
   pixelRatio: number;
 }
 
+export interface PickingInfoProps<DataT = any> {
```

Comment:
I prefer not overusing the name "props" especially in the context of the layer class. Maybe `GetPickingInfoParams`?